### PR TITLE
fix: offline PWA requires full login

### PR DESCRIPTION
**Pull request by jamil314**
## Description

Clearly describe what has been changed. Include relevant context or background.
Explain how the issue was fixed (if applicable) and the root cause.

Link this pull request to the GitHub issue (and optionally name the branch `ocrvs-<issue #>`)

## Checklist

- [ ] I have linked the correct Github issue under "Development"
- [ ] I have tested the changes locally, and written appropriate tests
- [ ] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [ ] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: packages/client/src/v2-events/features/events/useEvents/procedures/utils.ts [hunk 1]
```diff
@@ -167,10 +167,10 @@ export function setQueryDefaults<
...
   }
 ) {
-  queryClient.setQueryDefaults(
-    query.queryKey(),
-    options as Parameters<typeof queryClient.setQueryDefaults>[1]
-  )
+  queryClient.setQueryDefaults(query.queryKey(), {
+    ...(options as Parameters<typeof queryClient.setQueryDefaults>[1]),
+    networkMode: 'online'
+  })
```
Comment (Zangetsu101) [inline packages/client/src/v2-events/features/events/useEvents/procedures/utils.ts:179]: Let's add some comments here describing why this is needed

**Review by Zangetsu101 — APPROVED** [review]
